Epsilon.Host.Frontend/vite.config.ts

```diff
@@ -15,11 +15,17 @@ export default ({ mode }: { mode: string }): UserConfigExport => {
             },
         },
         server: {
-            https: {
-                cert: certificate ? readFileSync(certificate) : undefined,
-                key: key ? readFileSync(key) : undefined,
-            },
-            port: 8000,
+            // https: {
```

Comment:
If code is no longer need it please remove it.

Reply:
removed